examples/02_async/entries.ts

```diff
@@ -18,4 +18,14 @@ export default defineEntries(
       },
     };
   }
+  // getSsrConfig
+  // FIXME SSR streaming and RSC streaming conflict. Let's revisit it later.
```

Comment:
I think I understand what's happening. We need to send our `<script>` in the middle of SSR streaming. Let me see how we could do that in a simple way... 🤔

Reply:
#78